public/scripts/utils.js

```diff
@@ -44,4 +44,99 @@ module.exports = {
       .normalize('NFD')
       .replace(/[\u0300-\u036f]/g, '');
   },
+
+  sortByColors: function (colors) {
+    const isGray = (rgb, range) => {
+      var { r, g, b } = { ...rgb };
+      return (
+        r >= g - range &&
+        r <= g + range &&
+        b >= g - range &&
+        b <= g + range &&
+        r >= b - range &&
+        r <= b + range
+      );
+    };
+
+    const rgbToHsl = (rgb) => {
+      var { r, g, b } = { ...rgb };
+      // Make r, g, and b fractions of 1
```

Comment:
Isn't this called normalizing?

```suggestion
      // Normalize r, g, and b
```

If you choose this wording because _normalize_ is a "difficult" word, I don't think it helps anyone to describe it this way.